### DISCUSSION TITLE
Why some block content cannot be pasted with styles?

### DISCUSSION THREAD

**Discussion by gitacoco** [GitbookIO/community#787] [0. General bugs and questions]
Hi Gitbook Folks,

I have encountered several instances where content cannot be copied and pasted from one location to another with its original formatting intact. Although I’m certain the content is stored in RAM, it can only be pasted using “paste without formatting.” For example, here is a video recording showing the inability to move text in Quote blocks from the main body to tabs.

https://github.com/user-attachments/assets/f6c20a39-e8d2-464f-80a6-b81799fbfba1

Please feel free to reach out if you’d like more examples of scenarios that trigger this issue.

**Comment by emmerich** [comment]
Hi @gitacoco - thanks for raising this. We noticed the bug too, caused by a deployment this morning. A fix is on the way and will be deployed within the hour.
